src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -129,6 +129,11 @@ private static WebHostSettings GetDefaultSettings()
                 settings.SecretsPath = Path.Combine(home, @"data\Functions\secrets");
             }
 
+            if (string.IsNullOrEmpty(settings.ScriptPath))
+            {
+                throw new InvalidOperationException("Unable to determine function script root directory.");
```

Comment:
Wouldn't hurt to have error tell you to set `AzureWebJobsScriptRoot` to a folder.